bzt/modules/blazemeter.py

```diff
@@ -884,9 +884,18 @@ def resolve_account(self, account_name):
             if not account:
                 raise TaurusConfigError("BlazeMeter account not found by ID: %s" % account_name)
 
+        all_accounts = self.user.accounts()
```

Comment:
would be nice to avoid unnecessary api call. something like that:
```
        if account is None:
            self.user.fetch()
            all_accounts = self.user.accounts()
            for acc in all_accounts:
                if acc["owner"]["id"] == self.user['id']:
                    self.log.debug("Using userID matched account: %s", acc)
                    account = acc
                    break
            else:
                account = all_accounts.first()
                self.log.debug("Using first account: %s", account)

```